docs/gen-ai/llm-spans.md

```diff
@@ -36,49 +50,356 @@ Instrumentations that support it, MUST offer the ability to turn off capture of
+     | `gen_ai.system`     | `"openai"`                                            |
+     | Event payload (with sensitive content) | `{"index":0,"finish_reason":"tool_calls","message":{"role":"assistant","tool_calls":[{"id":"call_VSPygqKTWdrhaFErNvMV18Yl","function":{"name":"get_link_to_otel_semconv","arguments":"{\"semconv\":\"GenAI\"}"},"type":"function"}]}` |
+     | Event payload (withput sensitive content) | `{"index":0,"finish_reason":"tool_calls","message":{"role":"assistant","tool_calls":[{"id":"call_VSPygqKTWdrhaFErNvMV18Yl","function":{"name":"get_link_to_otel_semconv"},"type":"function"}]}` |
+
+2. Application executes the tool call. Application may create span which is not covered by this semantic convention.
+3. Final chat completion call
+
+   |   Attribute name    |                     Value                             |
+   |---------------------|-------------------------------------------------------|
+   | Span name           | `"chat.completion gpt-4"`                             |
+   | `gen_ai.system`     | `"openai"`                                            |
+   | `gen_ai.request.model`| `"gpt-4"`                                           |
+   | `gen_ai.request.max_tokens`| `200`                                          |
+   | `gen_ai.request.top_p`| `1.0`                                               |
+   | `gen_ai.response.id`| `"chatcmpl-call_VSPygqKTWdrhaFErNvMV18Yl"`            |
+   | `gen_ai.response.model`| `"gpt-4-0613"`                                     |
+   | `gen_ai.usage.completion_tokens`| `52`                                      |
+   | `gen_ai.usage.prompt_tokens`| `47`                                          |
+   | `gen_ai.response.finish_reason`| `"tool_calls"`                             |
+
+   Events parented to this span:
+   (in this example, the event content matches the original messages, but applications may also drop messages or change their content)
+
+   - `gen_ai.user.message` (not reported when capturing sensitive content is disabled)
+
+     |   Property          |                     Value                             |
+     |---------------------|-------------------------------------------------------|
+     | `gen_ai.system`     | `"openai"`                                            |
+     | Event payload       | `{"role":"user","content":"How to instrument GenAI library with OTel?"}` |
+
+   - `gen_ai.assistant.message` (could be reported when capturing sensitive content is disabled)
+
+     |   Property          |                     Value                             |
+     |---------------------|-------------------------------------------------------|
+     | `gen_ai.system`     | `"openai"`                                            |
+     | Event payload (with sensitive content) | `{"role":"assistant","tool_calls":[{"id":"call_VSPygqKTWdrhaFErNvMV18Yl","function":{"name":"get_link_to_otel_semconv","arguments":"{\"semconv\":\"GenAI\"}"},"type":"function"}]}` |
```

Comment:
The event payload here contains the role attribute in the stringified value, while in the definition the `role` is an attribute of itself. Do we need the `role` attribute?

Reply:
same here, no more role for now - we can always add it back later if needed